server/src/Icepeak/Server/Config.hs

```diff
@@ -112,6 +117,11 @@ configParser environment = Config
         metavar "WS-PING-INTERVAL" <>
         value 30 <>
         help "The interval of time (in seconds) between two pings to the WebSocket clients. It is instrumental in keeping connections alive.")
+  <*> option auto
+       (long "websocket-pong-timeout" <>
+        metavar "WS-PONG-TIMEOUT" <>
+        value 30 <>
+        help "The timespan in seconds after sending a ping during which the client has to respond with a pong. If no pong is sent within this timeout threshold then the connection is considered to have timed out and it will be terminated.")
```

Comment:
```suggestion
        help "The timespan in seconds after sending a ping during which the client has to respond with a pong. If no pong is sent within this timeframe then the connection is considered to have timed out and it will be terminated.")
```